init.lua

```diff
@@ -77,21 +77,100 @@ local function get_v(v)
 	return math.sqrt(v.x ^ 2 + v.z ^ 2)
 end
 
+local function ensure_passengers_exists(entity)
```

Comment:
For a PR on the vehicles mod thing I might need to make this a public part of the API. Do you like this name?

Reply:
Sounds good! 👍